tokens/connect/ledger.md

```diff
@@ -73,65 +85,70 @@ As of Chrome version 91, users that want to connect their Ledger device to MetaM
-![MetaMask Ledger Account Funded](/images/tokens/connect/ledger/ledger-6.png)
+Next, you will need to obtain some MOVR or DEV tokens and using the address you just copied, send the tokens to your account. After the transaction has successfully gone through, you will see your balance update.
+
+If you need DEV tokens for the Moonbase Alpha TestNet, you can head to the faucet to [Get Tokens](/builders/get-started/moonbase/#get-tokens).
+
+## Send Tokens {: #send-tokens } 
+
+Next up is sending and signing transactions on Moonbeam using your Ledger device. To get started sending a transaction, click on the **Send** button:
+
+![MetaMask Ledger Account Funded](/images/tokens/connect/ledger/ledger-8.png)
 
 As you would in a standard transaction, set the recipient address, enter the number of tokens to send, review transaction details and confirm it. This will initiate the transaction signature wizard in your Ledger device. Here, take the following steps:
 
  1. Click the button to proceed to the next screen. Your Ledger device is only warning you to review the transaction
- 2. Check the number of tokens being sent. Please note that the token corresponds to the network MetaMask is connected to. **In this case, it is DEV tokens and not ETH!** When ready, proceed to the next screen
+ 2. Check the number of tokens being sent then proceed to the next screen
  3. Check the recipient's address and proceed to the next screen
- 4. Check the chain ID of the network. This information confirms which network MetaMask is connected to. For example, for Moonbase Alpha, the chain ID is {{ networks.moonbase.chain_id }} (hex: {{ networks.moonbase.hex_chain_id}}), Moonriver {{ networks.moonriver.chain_id }} (hex: {{ networks.moonriver.hex_chain_id }}), and Moonbeam 1284 (not yet live). When ready, proceed to the next screen
+ 4. *This step only applies if you're using the Ethereum app*. Check the chain ID of the network. This information confirms which network MetaMask is connected to. For example, for Moonbase Alpha, the chain ID is 1287 (hex: 0x507), Moonriver 1285 (hex: 0x505), and Moonbeam 1284 (not yet live). When ready, proceed to the next screen
  5. Check the max fees applicable to this transaction. This is the gas price multiplied by the gas limit you've set on MetaMask. When ready, proceed to the next screen
  6. If you agree with all the transaction details, approve it. This will sign the transaction and will trigger MetaMask to send it. If not, proceed to the next screen
  7. If you don't agree with all the transaction details, reject it. This will cancel the transaction, and MetaMask will mark it as failed
 
-!!! note
-    At the time of writing, the token name is always shown as `ETH`. Please note that the token being handled is the one corresponding to the network MetaMask is connected to.
-
-![MetaMask Ledger Transaction Wizard](/images/tokens/connect/ledger/ledger-7.png)
+![MetaMask Ledger Transaction Wizard](/images/tokens/connect/ledger/ledger-9.png)
```

Comment:
This image shows (MOVR or DEV, NOT ETH!) This comment is valid only for the Ethereum App, not sure what you think

Reply:
I think it's valid for the Moonbeam app because you're not using ETH there either? I'm not sure... I'm open to changing it if you have any suggestions on what it should be instead... I'm just not sure what else it should be